src/admin/index.js

```diff
@@ -772,6 +775,280 @@ module.exports = ({
     })
   }
 
+  /**
+   * @param {array} ACL
+   * @return {Promise<void>}
+   */
+  const createAcls = async ({ acl }) => {
+    if (!acl || !Array.isArray(acl)) {
+      throw new KafkaJSNonRetriableError(`Invalid ACL array ${acl}`)
+    }
+    if (acl.length === 0) {
```

Comment:
Should we really throw on this? I think in other situations where the user basically calls us with a no-op (create 0 items) we just resolve.